docs/static/settings-file.asciidoc

```diff
@@ -134,6 +134,23 @@ guaranteed, but you save the processing cost of preserving order.
 
 | `auto`
 
+| `pipeline.ecs_compatibility`
+a|
+Sets the pipeline's default value for `ecs_compatibility`, a setting that is available to plugins that implement an ECS Compatibility mode for use with the Elastic Common
+Schema.
+Possible values are:
+
+* `disabled`
+* `v1`
+* `v2`
+
+This option allows the early opt-in (or preemptive opt-out) of ECS Compatibility modes in plugins,
```

Comment:
Consider adding a link to the doc goodness (<<ecs-ls>>) in #12830 after it is merged.